File: src/main/java/com/ject/studytrip/dummy/application/dto/CreateDummyTripCommand.java
package com.ject.studytrip.dummy.application.dto;

import com.ject.studytrip.trip.domain.model.TripCategory;
import java.time.LocalDate;

public record CreateDummyTripCommand(
        String name, String memo, TripCategory tripCategory, LocalDate endDate) {
    public static CreateDummyTripCommand of(
            String name, String memo, TripCategory tripCategory, LocalDate endDate) {
        return new CreateDummyTripCommand(name, memo, tripCategory, endDate);
    }
}

File: src/main/kotlin/com/ject/studytrip/dummy/application/dto/CreateDummyTripCommand.kt
package com.ject.studytrip.dummy.application.dto

import com.ject.studytrip.trip.domain.model.TripCategory
import java.time.LocalDate

data class CreateDummyTripCommand(
    val name: String,
    val memo: String,
    val tripCategory: TripCategory,
    val endDate: LocalDate?,
)
